examples/gsph/run_disc_1sink.py

```
@@ -0,0 +1,265 @@
+    res = glob.glob(dump_prefix + "*.sham")
+
+    f_max = ""
+    num_max = -1
+
+    for f in res:
+        try:
+            dump_num = int(f[len(dump_prefix) : -5])
+            if dump_num > num_max:
+                f_max = f
+                num_max = dump_num
+        except:
```

Ruff (0.16.2)

[error] Do not use bare `except`

(E722)